distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Extensions.enso

```diff
@@ -1791,6 +1786,35 @@ Text.parse_time_of_day self format:Date_Time_Formatter=Date_Time_Formatter.iso_t
 Text.parse_time_zone : Time_Zone ! Time_Error
 Text.parse_time_zone self = Time_Zone.parse self
 
+## ALIAS mid, slice, substring
+   GROUP Selections
+   Creates a new Text by selecting the specified range of the input.
+
+   Arguments:
+   'start' - the character position to start the substring at. 0-based index. Has support for negative values.
+   'length' - the number of characters to include from start.
```

Comment:
Please note to use consistent style for the `Arguments:` section, like other methods.

```suggestion
   - start: the character position to start the substring at. 0-based index. Has support for negative values.
   - length: the number of characters to include from start.
```